### PR TITLE
Resolves 404s on retired course

### PR DESCRIPTION
**Pull request by nyashamaphosa**
## Status

* Current Status: Ready for review
* Closes https://github.com/NCCE/teachcomputing.org-issues/issues/2338

## What's changed?

*Fix for 404 errors on retired courses*

### CODE REVIEW

**Review by dcorking — COMMENTED** [review]
Good work. I'd appreciate your feedback on the comments below, please.

**Comment by dcorking** [comment]
It also looks like the link shows for retired courses on the top part of a certificate dashboard such as http://teachcomputing.rpfdev.com/certificate/cs-accelerator (sorry for not spotting this in earlier review)
Screenshot 2023-04-24 at 12 37 50
I think these things live in a component, that you might find easiest to develop with using the previews for example http://teachcomputing.rpfdev.com/rails/components/course_activity_component/with_achievements

File: app/models/activity.rb [hunk 1]
```diff
@@ -54,4 +54,8 @@ def self.cs_accelerator_diagnostic_tool
   def online?
     category == ONLINE_CATEGORY
   end
+
+  def is_not_retired?
```
[the remark this answers is not in the document]
Comment (nyashamaphosa) [reply app/models/activity.rb:58]: active_at_stem? sounds great, i will change it to that thank you

**Comment by dcorking** [comment]
Nice fixes. It looks like you have missed line 12 of https://github.com/NCCE/teachcomputing.org/blob/ef26b684921ad4dfa65b621f2b419800efcb37a7/app/views/certificates/cs_accelerator/_achievements.html.erb#L12

**Pull request merged**